clients/tabby-agent/src/chat/smartApply.ts

```diff
@@ -49,15 +49,15 @@ export class SmartApplyFeature implements Feature {
     //nothing
   }
 
-  async provideSmartApplyEdit(params: SmartApplyCodeParams, token: CancellationToken): Promise<boolean> {
-    logger.info("Getting document");
+  async provideSmartApplyEdit(params: SmartApplyParams, token: CancellationToken): Promise<boolean> {
+    logger.debug("Getting document");
```

Comment:
Avoid logging too many messages at the `info` level.
I have changed them to the debug level.
When debugging, set the output channel log level to view different level logs.